### PR TITLE
Create monocker.md

### PR DESCRIPTION
**Pull request by ippocratis**
Added monocker to monitors

### CODE REVIEW

File: apps/monitors/monocker.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    container_name: monocker
+    image: petersem/monocker
+    environment:
+      SERVER_LABEL: 'Dev'
+      MESSAGE_PLATFORM: 'telegram@your_bot_id@your_chat_id'
+      # MESSAGE_PLATFORM: 'pushbullet@your_api_key@your_device_id'
+      # MESSAGE_PLATFORM: 'pushover@your_user_key@your_app_api_token'
+      # MESSAGE_PLATFORM: 'discord@webhook_url'
+      LABEL_ENABLE: 'false'
+      ONLY_OFFLINE_STATES: 'false''
```
Comment (tborychowski) [inline apps/monitors/monocker.md]: too many apostrophes?

File: apps/monitors/monocker.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      EXCLUDE_EXITED: 'false'.
```
Comment (tborychowski) [inline apps/monitors/monocker.md]: the dot at the end probably shouldn't be there?

**Review by tborychowski — CHANGES_REQUESTED** [review]

File: apps/monitors/monocker.md (hunk 2 above)
Comment (ippocratis) [reply apps/monitors/monocker.md]: Deleted the extra apostrophe and the dot

**Review by tborychowski — APPROVED** [review]

**Comment by tborychowski** [comment]
thank you!

**Pull request merged**